articles/custom_storage_location.md

```diff
@@ -457,6 +375,43 @@ projectDestination <- synRestPOST('/projectSettings', body=toJSON(projectDestina
 
 Navigate to your **Project/Folder -> Tools -> Change Storage Location**. In the resulting pop-up, select the **Google Cloud Storage Bucket** option and fill in the relevant information, where Bucket is the name of your external bucket, Base Key is the name of the folder in your bucket to upload to, and Banner is a short description such as who owns the storage location.
 
+### Adding Files in Your Google Cloud Bucket to Synapse
+
+If your bucket is set for read-write access, files can be added to the bucket using the standard Synapse interface (web or programmatic). 
+
+If the bucket is read-only or you already have content in the bucket, you will have to add representations of the files in Synapse programmatically. This is done using a `FileHandle`, which is a Synapse representation of the file. 
```

Comment:
```suggestion
If the bucket is read-only, or you already have content in the bucket, you will first create representations of the files in Synapse programmatically by creating a Synapse `File` to store a `dataFileHandleId`. 
```

Reply:
I see we do not define file handles anywhere in the documentation. As a follow-up, will create content describing this Synapse feature https://github.com/Sage-Bionetworks/synapseDocs/issues/743. This does not block merging this PR.